### PR TITLE
 [#112]Corrected skip methods behavior of JsonParserImpl; 

### PR DESCRIPTION
**Pull request by api-from-the-ion**
Corrected the behavior of the skip methods in the parser implementation

### CODE REVIEW

**Comment by api-from-the-ion** [comment]
I'm writing the tests here, and I'm missing Junit 5 dearly. My question: could we move to Junit 5? This should be just a simple work of changing dependencies and modifying the 380 tests to use annotations and so on.

File: impl/src/test/java/org/eclipse/parsson/JsonParserFixture.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+import jakarta.json.Json;
+import jakarta.json.JsonArray;
+import jakarta.json.JsonObject;
+import jakarta.json.stream.JsonParser;
+
+/**
+ * Class with methods that creates JsonParser with different configuration from different sources and runs the test code with this parser
+ */
+public class JsonParserFixture {
```
Comment (jansupol) [inline impl/src/test/java/org/eclipse/parsson/JsonParserFixture.java:31]: That is a self-explanatory name :(

**Review by lukasj — CHANGES_REQUESTED** [review]
* avoid whitespace changes - use spaces and not tabs
* avoid static imports - they just make reviewers life more difficult when mixed with other changes
* are pom updates required here? they are usually handled through separate PRs (or through PRs which are to be just rebased instead of being squashed)

**Comment by lukasj** [comment]
> My question: could we move to Junit 5? This should be just a simple work of changing dependencies and modifying the 380 tests to use annotations and so on.

If someone provides PR for this, I see no reason to not move there.

File: impl/src/test/java/org/eclipse/parsson/JsonParserFixture.java (hunk 1 above)
Comment (api-from-the-ion) [reply impl/src/test/java/org/eclipse/parsson/JsonParserFixture.java:31]: Yes, I know. ;-) But what else should I write here? This is a fixture class for the tests involving the parser, so many of them. I don't like to repeat this try-with-resource line over and over again.

Or did you mean like we don't need the JavaDoc for the class itself, only the methods of this particular class? I mean, it's maybe not necessary, but this comment also doesn't hurt anybody, right? ;-)

**Comment by api-from-the-ion** [comment]
> * avoid whitespace changes - use spaces and not tabs
> 
>     * avoid static imports - they just make reviewers life more difficult when mixed with other changes
> 
>     * are pom updates required here? they are usually handled through separate PRs (or through PRs which are to be just rebased instead of being squashed)

- I'll convert them. Yasson using the Checkstyle rule for this - shouldn't it be activated here also?

- I remove the static imports and make a commit
- Should I create a new branch with POM changes and cherry-pick these changes over there? I forgot to create a branch for my changes and done it on the changes on the master directly. ;-(

**Comment by api-from-the-ion** [comment]
> If someone provides PR for this, I see no reason to not to move there.

OK, so we don't need some ticket for this? Just the PR full of commits to review? And another question: should I create a branch from the current master or is it OK, to fork some of the branches?

**Review by lukasj — APPROVED** [review]

**Pull request merged**